include/Foundation/NSException.h

```diff
@@ -46,8 +46,8 @@ FOUNDATION_EXPORT NSString* const NSOverflowException;
 FOUNDATION_EXPORT_CLASS
 @interface NSException : NSObject <NSCoding, NSCopying>
 + (NSException*)exceptionWithName:(NSString*)name reason:(NSString*)reason userInfo:(NSDictionary*)userInfo;
-+ (void)raise:(NSString*)name format:(NSString*)format, ...;
-+ (void)raise:(NSString*)name format:(NSString*)format arguments:(va_list)argList;
++ (void)raise:(NSString*)name format:(NSString*)format, ... NS_FORMAT_FUNCTION(2, 3);
++ (void)raise:(NSString*)name format:(NSString*)format arguments:(va_list)argList NS_FORMAT_FUNCTION(2, 0);
```

Comment:
since it takes a `va_arg`, the compiler cannot tell us anything about the arguments -- we still want the format string security check though!